pkg/gitreceive/k8s_util.go

```diff
@@ -241,7 +241,6 @@ func progress(msg string, interval time.Duration) chan bool {
 				return
 			case <-tick:
 				fmt.Println(msg)
-			default:
```

Comment:
what's the purpose of taking out this `default:`? Also, how is it related to this PR?